src/components/viewer/DrawTools.vue

```diff
@@ -783,23 +783,38 @@ export default {
       this.copiedAnnot = feature.properties.annot.clone();
       this.$notify({type: 'success', text: this.$t('notif-success-annotation-copy')});
     },
-    convertLocation(originalLocation, image) {
+    async convertLocation(copiedAnnot, destImage) {
       /* If we want to paste in the same image but in another slice */
-      if (this.image.id === this.copiedAnnot.image) {
-        return this.copiedAnnot.location;
+      if (destImage.id === copiedAnnot.image) {
+        return copiedAnnot.location;
       }
 
-      let geometry = new WKT().readGeometry(originalLocation);
-      let wrapper = this.imageWrapper;
+      /* Compute the rescaling factor if the resolution is known for both images */
+      let scale = 1;
+      let srcImage = await ImageInstance.fetch(copiedAnnot.image);
+      if (srcImage.physicalSizeX !== null && destImage.physicalSizeX !== null) {
+        scale = srcImage.physicalSizeX / destImage.physicalSizeX;
```

Comment:
Scaling along Y axis that can be different, computed from the `physicalSizeY`. If not set, `physicalSizeX` should be used for X & Y.

Reply:
The computation of the scaling factor now takes into account the `physicalSizeY` if it is set.
Changed in addec27